src/importexport/musicxml/internal/musicxml/export/exportmusicxml.cpp

```diff
@@ -3274,8 +3274,6 @@ static String symIdToOrnam(const SymId sid)
         return String(u"other-ornament smufl=\"%1\"").arg(String::fromAscii(name.ascii()));
         break;
```

Comment:
This cannot be reached as well.

```suggestion
```

Reply:
True, but no compiler complains about it

Reply:
And all the cases above have the same issue, so 'Od rather keep it as is